vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/text_processing/README.md

```diff
@@ -1,3 +1,7 @@
 # Machine Learning Text Processing Pipeline
 
-Please add to it!
+Pipeline instructions specifically related to processing model input text.
+
+## Embedding Processing
+
+explain og:title... (and in other places)
```

Comment:
Please add an explanation of what the processor does with regards to og:title and provide a link to Facebook documentation.

Also, please add an explanation to other README files where we use og:title